src/astmanproxy.c

```diff
@@ -720,8 +725,8 @@ int main(int argc, char *argv[])
 		exit(1);
 	}
 
-	/* If we are not in debug mode, then fork to background */
-	if (!debug) {
+	/* If we are not in foreground mode, then fork to background */
+	if (!foreground) {
```

Comment:
@simonl2002 Question:
1. So we won't get any logging now with foreground mode, intentional?
2. If we ever need to run it in debug mode, it'll run in background and the logs won't go to the console.

Reply:
yeah we'd have to run with -f -d